rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -85,7 +85,7 @@ def test_ppo_compilation_and_schedule_mixins(self):
                 enable_connectors=True,
             )
             .callbacks(MyCallbacks)
-            .experimental(_enable_rl_module_api=True)
+            .rl_module()
```

Comment:
no reason to omit ``_enable_rl_module_api=True`` man.
just specify it to make things clearer.

Reply:
that's the point. You should not specify it.